pkg/vulnsrc/vulnerability/vulnerability.go

```diff
@@ -51,7 +51,13 @@ func FillAndFilter(vulns []DetectedVulnerability, severities []Severity, ignoreU
 		if vulnerabilities[i].PkgName != vulnerabilities[j].PkgName {
```

Comment:
Should an `Unknown` severity be ranked higher than a known `Low` one, if the `Low` Vulnerability has a `PkgName` set? I see this is the current behaviour but I'm just probing to see if it is what we want. You can see a test that demonstrates this here: https://github.com/aquasecurity/trivy/pull/201/commits/895b7312075eb8358a06ac5b5cea50dffbc1e49f

Reply:
Basically, all vulnerabilities have PkgName. I'd like to display vulnerabilities sorted by a package name. Why do you expect `hogepkg`? This package doesn't seem to be defined.
https://github.com/aquasecurity/trivy/commit/895b7312075eb8358a06ac5b5cea50dffbc1e49f#diff-bc5f6ed1d12bdab4d1aeca18ca3003d3R90

Reply:
Makes sense if PkgName takes preference!